core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -294,4 +318,78 @@ protected boolean runSerially() {
             return false;
         }
     }
+     
+	public void backupTable(CairoSecurityContext securityContext, CharSequence tableName, Path path) {
+		if (null == configuration.getBackupRoot()) {
+			throw CairoException.instance(0).put("Backup is disabled, no backup root directory is configured in the server configuration ['cairo.sql.backup.root' property]");
+		}
+		
+		try (TableReader reader = getReader(securityContext, tableName)) {
+			cloneMetaData(tableName, configuration.getFilesFacade(), path, configuration.getRoot(), configuration.getBackupRoot(), configuration.getMkDirMode());
+			RecordMetadata metaData = reader.getMetadata();
+			int timestampCol = metaData.getTimestampIndex();
+
+			try (TableWriter backupWriter = getBackupWriter(securityContext, tableName); RecordCursor cursor = reader.getCursor()) {
+				Record record = cursor.getRecord();
+				while (cursor.hasNext()) {
+					TableWriter.Row row;
```

Comment:
Still need to figure out how to use 
RecordToRowCopier copier = assembleRecordToRowCopier(asm, record, metadata, listColumnFilter);